libraries/Microsoft.Bot.Builder.Ai.LUIS/Microsoft.Bot.Builder.Ai.LUIS.csproj

```diff
@@ -53,7 +53,7 @@
 		<PackageReference Include="Microsoft.Bot.Builder.Core" Condition=" '$(PackageVersion)' == '' " Version="4.0.0-local" />
 		<PackageReference Include="Microsoft.Bot.Builder.Core" Condition=" '$(PackageVersion)' != '' " Version="$(PackageVersion)" />
 		<PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.10" />
-		<PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+		<PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```

Comment:
This is a fairly big change, with impact beyond the obvious. Staying aligned the version of Newtonsoft.json across out dependencies and all of our projects + samples is more complex than it may seem. 

Could you roll this particular change back?

Reply:
Ok. I'll do this as soon as possible.